apps/web-giddh/src/app/invoice/create/invoice.create.component.scss

```diff
@@ -23,7 +23,7 @@
     width: 100%;
     height: 80px;
     padding: 20px 0;
-    background-color: "$veryLightGray";
+    background-color: "$very-light-gray";
```

Comment:
why are we using double quotes here?

Reply:
i am not using here , already added

Reply:
Please remove double quotes from here even if you have not added as it is incorrect